packages/tabs/src/styled/StyledTabs.ts

```diff
@@ -20,26 +19,64 @@ interface IStyledTabsProps {
   isVertical?: boolean;
 }
 
+const verticalStyling = ({ theme }: ThemeProps<DefaultTheme>) => {
+  return css`
+    display: table;
+
+    ${StyledTabList} {
+      display: table-cell;
+      margin-bottom: 0;
+      border-bottom: none;
+      vertical-align: top;
+    }
+
+    ${StyledTab} {
+      display: block;
+      margin-bottom: ${theme.space.base * 5}px;
+      margin-left: ${theme.rtl && '0'};
+      border-left: ${theme.rtl && '0'};
+      border-right-style: ${theme.rtl && theme.borderStyles.solid};
+      border-bottom-style: none;
+      border-left-style: ${!theme.rtl && theme.borderStyles.solid};
```

Comment:
My preference would be to use constructs similar to what you have below throughout this block. i.e.

```css
border-${theme.rtl ? 'right' : 'left'}-style: ${theme.borderStyles.solid};
```

It improves scanability for trying to figure out how the logic is going to trigger one property over another. And it starts to set a pattern precedent in our code for "this is an RTL-manipulated CSS property".